src/application/common/command.ts

```diff
@@ -15,9 +15,9 @@ export interface Command {
     iconClass?: string;
 }
 export interface CommandHandler {
-    execute(): any;
-    isEnabled(): boolean;
-    isVisible?(): boolean;
+    execute(...args: any[]): any;
+    isEnabled(...args: any[]): boolean;
```

Comment:
I'd rather make `isEnabled` optional and then not overload the functions below with the additional function signature.